### DISCUSSION TITLE
Syntax error for optional chaining operator

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [eslint/eslint#18816] [Q&A] [answered]
Hi! I don't get why I get `Parsing error: Unexpected token .` on [this line](https://github.com/EmilyGraceSeville7cf/fish-tui-gum/blob/main/src/types.js#L30), though the node version specified in [package.json](https://github.com/EmilyGraceSeville7cf/fish-tui-gum/blob/main/package.json#L15) should support `?.` operator AFAIK.

**Comment by aladdin-add** [comment, marked as answer]
you need to config `ecmaVersion` >= 2020 (optional chaining was introduced in es2020).
